pkg/admission/karydia/admission.go

```diff
@@ -35,10 +35,16 @@ var kindServiceAccount = metav1.GroupVersionKind{Group: "", Version: "v1", Kind:
 type KarydiaAdmission struct {
 	logger        *logrus.Logger
 	kubeClientset *kubernetes.Clientset
+	karydiaConfig *v1alpha1.KarydiaConfig
+}
+func (k *KarydiaAdmission) UpdateConfig(karydiaConfig v1alpha1.KarydiaConfig) error {
+	k.karydiaConfig = karydiaConfig.DeepCopy()
```

Comment:
Not sure, but is a DeepCopy actually necessary? Isn't it possible to assign the value directly?